src/index.js

```diff
@@ -35,144 +32,23 @@ export { default as ZoomMouseWheelTool } from './tools/ZoomMouseWheelTool.js';
-export { default as import } from './thirdParty/import.js';
-
-// END V3 EXPORTS
-
-export { drawing };
-
-export { default as external } from './externalModules.js';
-export { default as EVENTS } from './events.js';
-
 export { default as referenceLines } from './referenceLines/index.js';
 export { default as orientation } from './orientation/index.js';
 
-export {
-  default as requestPoolManager
-} from './requestPool/requestPoolManager.js';
-
-export {
-  default as setContextToDisplayFontSize
-} from './util/setContextToDisplayFontSize.js';
-export { default as scrollToIndex } from './util/scrollToIndex.js';
-export { default as scroll } from './util/scroll.js';
-export { default as roundToDecimal } from './util/roundToDecimal.js';
-export {
-  projectPatientPointToImagePlane,
-  imagePointToPatientPoint,
-  planePlaneIntersection
-} from './util/pointProjector.js';
-
-export {
-  default as pointInsideBoundingBox
-} from './util/pointInsideBoundingBox.js';
-export { default as pointInEllipse } from './util/pointInEllipse.js';
-export { default as makeUnselectable } from './util/makeUnselectable.js';
-export { default as getRGBPixels } from './util/getRGBPixels.js';
-export {
-  getDefaultSimultaneousRequests,
-  getMaxSimultaneousRequests,
-  getBrowserInfo,
-  isMobileDevice
-} from './util/getMaxSimultaneousRequests.js';
-
-export { default as getLuminance } from './util/getLuminance.js';
-export { default as drawTextBox } from './util/drawTextBox.js';
-export { default as drawEllipse } from './util/drawEllipse.js';
-export { default as drawCircle } from './util/drawCircle.js';
-export { default as drawArrow } from './util/drawArrow.js';
-export { default as copyPoints } from './util/copyPoints.js';
-export { default as calculateSUV } from './util/calculateSUV.js';
-export {
-  default as calculateEllipseStatistics
-} from './util/calculateEllipseStatistics.js';
-
-export {
-  default as wwwcSynchronizer
-} from './synchronization/wwwcSynchronizer.js';
-export {
-  default as updateImageSynchronizer
-} from './synchronization/updateImageSynchronizer.js';
-export { default as Synchronizer } from './synchronization/Synchronizer.js';
-export {
-  default as stackScrollSynchronizer
-} from './synchronization/stackScrollSynchronizer.js';
-export {
-  default as stackImagePositionSynchronizer
-} from './synchronization/stackImagePositionSynchronizer.js';
-export {
-  default as stackImagePositionOffsetSynchronizer
-} from './synchronization/stackImagePositionOffsetSynchronizer.js';
-export {
-  default as stackImageIndexSynchronizer
-} from './synchronization/stackImageIndexSynchronizer.js';
-export {
-  default as panZoomSynchronizer
-} from './synchronization/panZoomSynchronizer.js';
-
-export { default as toolStyle } from './stateManagement/toolStyle.js';
-export {
-  addToolState,
-  getToolState,
-  removeToolState,
-  clearToolState,
-  setElementToolStateManager,
-  getElementToolStateManager
-} from './stateManagement/toolState.js';
```

Comment:
I think these make sense as internal, but we may want to identify _abstractions_ that should exist to replace functionality like this that we're taking away? restoring/clearing state feels like something you might want to do for a still fairly simple implementation, but I'm not 100% sure these methods are the best way to expose that.

Reply:
Currently these haven't been removed, just moved to the import interface. I'll leave them 'as is' for now, but perhaps we should abstract them further down the road 👍 .